src/Driver/Connection.php

```diff
@@ -57,8 +57,6 @@ public function exec(string $sql): int|string;
      * previous statement. DO NOT RELY ON THIS BEHAVIOR which is driver-dependent: always use getLastInsertId() right
```

Comment:
Is this a typo? or where is `getLastInsertId()` defined?

Reply:
Looks like a typo to me.